CMakeLists.txt

```diff
@@ -41,10 +41,12 @@ set(EMCC_ARGS
   -O3
   -s ALLOW_MEMORY_GROWTH=${ALLOW_MEMORY_GROWTH}
   -s EXPORTED_RUNTIME_METHODS=["UTF8ToString"]
+  -s EXTRA_EXPORTED_RUNTIME_METHODS=["addFunction"]
   -s EXPORT_NAME="Ammo"
   -s MODULARIZE=1
   -s NO_EXIT_RUNTIME=1
   -s NO_FILESYSTEM=1
+  -s RESERVED_FUNCTION_POINTERS=20
```

Comment:
this should not be needed anymore. instead use `-s ALLOW_TABLE_GROWTH`

Reply:
Sounds good- should we do that for wasm and javascript builds?